### ISSUE TITLE
Library problématique avec les étudiants avec un skill à 5 (maxé)

### ISSUE THREAD

**Issue by Simon-Lessard** [JulienNigon/SpellcasterUniversity-Bug-Report#885]
Les étudiants avec un skill maxé qui vont à la librairie perdent leur temps car leur skil est déjà maxé et empêchent les autres étudiant d'utiliser la pièce. Il serait judicieux que les étudiants avec un skill maxé montent plutôt leur prochain meilleur skill lorsqu'ils ont des skill maxés. Sinon, si considéré un souci en terme de balance, il faudrait au moins que ces étudiants évitent la librairie.

**Comment by MlleMau** [comment]
Bonjour Simon-Lessard,

La bibliothèque n'est pas non plus rendue inutile, car cela leur permet de générer du mana. Cependant, les étudiants qui continuent d'aller dans un cours qu'ils ont déjà maxé ont déjà été signalés (mais je ne retrouve plus le topic en question, zut !)

**Comment by Neelan22** [comment]
Oui, c'est une problématique centrale en ce moment dans nos réflexions. On ne sait pas encore comment on va le modifier, mais il y aura une refonte à ce niveau là. De manière générale, l'IA sera améliorée. Ce n'est pas quelque chose qui se fait rapidement, mais nous n'oublions pas vos retours ! :)
